modules/nw-ptp-introduction.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-ptp.adoc
+
+[id="ptp-introduction_{context}"]
+= Introduction to PTP
+
+The Precision Time Protocol (PTP) is a protocol used to synchronize clocks in a network. When used in conjunction with hardware support, PTP is capable of sub-microsecond accuracy, which is far more accurate than Network Time Protocol (NTP).
+
+The linuxptp package includes the `ptp4l` and `phc2sys` programs for clock synchronization. The `ptp4l` program implements the PTP boundary clock and ordinary clock. The `ptp4l` program synchronizes the PTP hardware clock to the source clock with hardware time stamping and synchronizes the system clock to the source clock with software time stamping. The `phc2sys` program is only needed for hardware time stamping to synchronize the system clock to the PTP hardware clock on the network interface card (NIC).
+
+[id="ptp-elements_{context}"]
+== Elements of a PTP domain
+
+PTP included multiple nodes connected through the network, each representing a clock. The following type of clocks can be included in configurations:
+
+Grandmaster clock:: The grandmaster clock is the reference clock that provides standard time information to other clocks across the network.  The grandmaster clock needs to be able to ensure accurate and stable synchronisation. The grandmaster clock writes time stamps and responds to time requests from other clocks.
+
+Ordinary clock:: The ordinary clock has a single port connection that can play the role of source or destination clock, depending on its position in the network. The ordinary clock can read and write time stamps.
+
+Boundary clock:: The boundary clock has ports in two or more communication paths and can be a source and a destination to other destination clocks at the same time. The boundary clock works as a destination clock upstream: the destination clock receives the timing message, adjusts for delay, and then creates a new source time signal to pass down the network. The boundary clock produces a new timing packet that is still correctly synced with the source clock, and can reduce the number of connected devices reporting directly to the source clock. 
+
+[id="ptp-advantages-over-ntp_{context}"]
+== Advantages of PTP over NTP
+One of the main advantages that PTP has over NTP is hardware support present in various network interface controllers (NIC) and network switches. The specialized hardware allows PTP to account for delays in message transfer, and improves the accuracy of time synchronization. It is possible to use non-PTP enabled hardware components within the network. The use of non-PTP enabled hardware can often cause synchronization inaccuracies, which increases with every non-PTP enabled hardware used in the communication path. To achieve the best possible accuracy, it is recommended that all networking components between PTP clocks are PTP hardware enabled.
```

Comment:
PTP is supported in various network interface controllers (NIC) and network switches. The specialized hardware allows PTP to account for delays in message transfer, and improves the accuracy of time synchronization. ~It is possible to use non-PTP enabled hardware components within the network. The use of non-PTP enabled hardware can often cause synchronization inaccuracies, which increases with every non-PTP enabled hardware used in the communication path.~ To achieve the best possible accuracy, it is recommended that all networking components between PTP clocks are PTP hardware enabled.

Reply:
I would be interested in the reasoning behind this suggestion. 
My logic: being able to use non-PTP enabled hardware in the network seems to be an advantage to me, but we have to inform the reader that it has its limitations.

Reply:
I think I would agree with what @aireilly has suggested. It is a good idea to promote the positives and the line states that "To achieve the best possible accuracy, it is recommended that all networking components between PTP clocks are PTP hardware enabled."

Reply:
@kquinn1204 Thanks for the explanation! I'll remove it